### PR TITLE
Fixed API functions

### PR DESCRIPTION
**Pull request by afamiglietti**
At least, I think I did. Let me know what you think of my solution. 

I added all of the setEndPoints to the getEndPoints array, since the code mapping the getEndPoints already had functions to handle those endpoints.

I fixed a couple of what I think were typos where req was being used instead of req.query 

For the setStylesForPad method we'll need to update the API documentation. The problem was with passing in arrays as a parameter. The method documented in the API doc wasn't working because the value of any single parameter gets passed to the function by Express as a string. However, if the same parameter is included multiple times with multiple values, Express handily builds an array and passes them off. In other words: 

http://localhost:9001/pluginAPI/customStyles.styles.setStylesForPad?apikey=seekrit&styleIds=["style1", "style2"]&padId=stylesTest

wouldn't work because styleId would end up a string "["style1","style2"]" 

However if we use:

http://localhost:9001/pluginAPI/customStyles.styles.setStylesForPad?apikey=seekrit&styleIds=style1&styleIds=style2&padId=stylesTest

Then Express gives us ["style1","style2"] for req.styleIds and everything works. 

For the case in which only a single styleIds parameter is used, I wrote a little test to see if req.styleIds is a string, and if it is, wraps it in an array. 

Tell me what you think!
- Andy

### CODE REVIEW

File: customStyles.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ exports.customStyles = {
   endpoints: ["customStyles.styles.new","customStyles.styles.update", "customStyles.styles.globalDisable", "customStyles.styles.disable", "customStyles.styles.delete", "customStyles.styles.get", "customStyles.styles.stylesForPad", "customStyles.styles.allStyles", "customStyles.styles.disabledStyles"],
   styles: {
     new: function(styleId, css, padId, cb){
+      console.warn("New style:", styleId, css, padId);
```
Comment (JohnMcLear) [inline customStyles.js]: Did you mean to leave this warn in?

**Comment by JohnMcLear** [comment]
Looks good, I'd also include that ep.json post thing that I suggested, we discovered it can help in some cases.

**Comment by afamiglietti** [comment]
Awesome, I removed that console warn, thanks for catching that. The ep.json post line is in there.

**Comment by JohnMcLear** [comment]
not seeing the ep.json change in your commits..  Perhaps you didn't commit that file?

**Comment by JohnMcLear** [comment]
Oh I already commited it!

**Comment by JohnMcLear** [comment]
tnx

**Comment by afamiglietti** [comment]
Awesome, glad to help. I'm working on trying to integrate etherpad-lite into a classroom software project, so I may have more questions/contributions over the next couple of months...

**Comment by JohnMcLear** [comment]
Nice!  Edu is a good use case.

Disclaimer: I founded Primary Technology, the company behind http://primarypad.com :)  I don't work there any more though

**Comment by afamiglietti** [comment]
Neat! We're http://www.gomarca.com/ and we're mostly going for the University market in the US, so we're probably not direct competition :)

**Comment by JohnMcLear** [comment]
Looks really cool, you guys using ep_comments_page at all? We're doing a bunch of work on that the moment so it's a pretty exciting plugin to be using.

**Comment by JohnMcLear** [comment]
Looks like you guys have got link-unlink functionality too? http://www.gomarca.com/wp-content/uploads/2014/01/Screen-Shot-2014-09-23-at-11.56.29-AM.png - Thinking about contributing that back to the community as a plugin?

**Comment by afamiglietti** [comment]
Everything you're looking at in those screenshots is our ck-editor version, I literally sat down to start working on getting us onto etherpad on Monday :) Anything I develop along the way I'll contribute back.

Our codebase is https://github.com/calliopeinitiative/marca btw

By link-unlink functionality, do you mean adding and removing hyperlinks to a doc via the editor? If so, I'll probably take a stab at that plugin at some point, if nobody has yet, yeah.

**Comment by afamiglietti** [comment]
Oh, and I was planning on using the comments plugin, yeah. Good to know its in active development.

**Comment by JohnMcLear** [comment]
Yeah :)

On 3 Jun 2015 17:15, Andrew Famiglietti notifications@github.com wrote:

Everything you're looking at in those screenshots is our ck-editor version, I literally sat down to start working on getting us onto etherpad on Monday :) Anything I develop along the way I'll contribute back.

Our codebase is https://github.com/calliopeinitiative/marca btw

By link-unlink functionality, do you mean adding and removing hyperlinks to a doc via the editor? If so, I'll probably take a stab at that plugin at some point, if nobody has yet, yeah.

**Pull request merged**